src/qml/components/AboutOptions.qml

```diff
@@ -12,25 +12,25 @@ ColumnLayout {
     Information {
         Layout.fillWidth: true
         header:  qsTr("Website")
-        description: qsTr("bitcoincore.org >")
+        description: "bitcoincore.org >"
```

Comment:
while here, and working on a follow-up on this file, please seperate out the greater than symbol here. This shouldn't use a text symbol to represent the `right caret` icon as shown in the designs. the `right caret` icon is already in the repo.

Reply:
done in #144

Reply:
Great! Thanks, Jarol!